doc/source/parameters/cfd/dynamic_flow_control.rst

```diff
@@ -1,39 +1,56 @@
 
-.. warning::
+* The ``enable beta particle`` parameter is set to ``true`` to apply a proportional beta force on particles.
 
-  This value should be negative if imposed on an the inlet boundary, and positive if imposed on an outlet boundary.
+* The ``average velocity`` parameter specifies the targeted average velocity (:math:`m/s`). The value is compared to the calculated value at a boundary (for CFD solvers) or the whole domain (for CFD-DEM solver).
 
 .. note::
 
-  If you are simulating a problem with periodic boundary conditions at inlet and outlet walls, it is important to control the volumetric flow rate specifying the boundary id that corresponds to the inlet only (where the flow would have a negative value).
+  If you are simulating a problem with periodic boundary conditions, it is important to control the average velocity by specifying the boundary id that corresponds to the inlet only.
 
-* The ``boundary id`` parameter is the wall boundary where the flow rate intended is controlled (inlet or outlet).
+* The ``inlet boundary id`` parameter is the boundary where the average velocity intended is controlled (for CFD solvers only).
 
 * The ``flow direction`` parameter sets the absolute direction. It should be ``0`` if the flow is in the :math:`x` direction, ``1`` in the :math:`y` direction and ``2`` in the :math:`z` direction.
 
-* The ``initial beta`` parameter sets an initial :math:`\beta` coefficient that may speed up the convergence of the flow rate targeted. Some tests should be done to find a good initial :math:`\beta` coefficient.
+* The ``initial beta`` parameter sets an initial :math:`\beta` coefficient that may speed up the convergence of the average velocity target. Some tests should be done to find a good initial :math:`\beta` coefficient.
 
 .. tip:: 
 
-  A good method to find a reasonable initial beta is to test two or three different initial beta parameters, write down the given flow rate at the first time step in the simulation and do a regression. The correlation is linear and giving a proper value will greatly speed up the convergence.
+  A good method to find a reasonable initial beta is to test two or three different initial beta parameters, write down the given flow rate at the first time step in the simulation and do a regression. The correlation is linear and giving a proper value will reduce the time it takes to reach the target velocity.
+
+* The ``alpha`` parameter is a relaxation coefficient that is used to control the convergence speed or stability. The higher the value, the faster the convergence. However, if the value is too high, the convergence may be unstable.
+
+* The ``beta threshold`` parameter is the threshold of the previous calculated beta force where the new calculated beta doesn't need to be updated. This prevents the reassembly of the matrix because of the force term when reuse matrix is used for the non-linear solver
```

Comment:
I don't really understand this explanation. The beta threshold is the old beta, or is it a tolerance in the sense that if abs(old_beta-new_beta)<tolerance, we reuse the same matrix?

Reply:
`beta_threshold` is a threshold on the `beta_n` value (for instance 0.01 is 1%). The way it works is that `beta_n+1` is calculated as usual but if `abs(beta_n - beta_n+1) < abs(beta_n * beta_threshold)`, it changes the value of `beta_n+1` to `beta_n`. That prevent to update the source term (the beta force is added to force in scratch data I think). It doesn't means the same matrix is going to be reuse with the option enable (this is independent of the non-linear solver parameters), but it could help to do so since we force the code to keep the same value. I will try to reformulate this explaination

Reply:
I reformulated the doc if you want to look at it :)

Reply:
I understand better with your explanation, thanks. I think the inequality should be there as it really helps to understand (added as suggestion)